v3.0/tidb-in-kubernetes/get-started/deploy-tidb-from-kubernetes-minikube.md

```diff
@@ -102,7 +103,7 @@ kubectl -n kube-system get pods -l app=helm
 
 ### Add Helm repo
 
-Helm repo (https://charts.pingcap.org/) houses PingCAP managed charts, such as tidb-operator, tidb-cluster and tidb-backup, etc. Add and check the repo with following commands:
+Helm repo (`https://charts.pingcap.org/`) houses PingCAP managed charts, such as tidb-operator, tidb-cluster and tidb-backup, etc. Add and check the repo with following commands:
```

Comment:
Do we need to show the url explicitly?

Reply:
@dcalvin I'm afraid not. This link will be used in the command, not a direct click by users.